mlir/utils/jenkins/MIOpenDriver.py

```diff
@@ -549,15 +535,20 @@ def main(args=None):
         help="Only tune the MLIR kernels"
     )
     mutex_arg_group.add_argument(
-        "-b",
+        "-b", "--batch",
```

Comment:
How about --batch_mlir here ? and do we need just the -b because we have two long arguments starting with b, I think if possible we should drop the -b.

Reply:
I got rid of the long arguments that start with `-b` in most cases
But `--batch_mlir` is good, I'll change it

I'd weakly prefer keeping it because `./bin/MIOpenDriver.py -b -o=blah.csv` is the most common manual use case in my experience

Reply:
Ack.